application/services/responses/store_responses.rb

```diff
@@ -43,7 +43,7 @@ def create_response_entities_arr(input)
         input[:pages].each do |page|
           page_index = page.index
           page.items.each do |item|
-            next if item.type == 'Description' || item.type == 'Section Title' || item.type == 'Divider' || item.type == 'Jump to page'
+            next if item.type == 'Description' || item.type == 'Section Title' || item.type == 'Divider' || item.type == 'Jump to page' || item.type == 'Disable previous page'
```

Comment:
could define the strings as part of an array and use `.include?`.
Example: `['hello', 'world'].include? 'hello'  # => true`